Doc/includes/sqlite3/insert_more_people.py

```diff
@@ -4,13 +4,13 @@
 
 cur = con.cursor()
 
```

Comment:
Name of the file is now inconsistent :)

Reply:
Hah, good catch! I didn't think of that 😅

Reply:
BTW, this file is not referenced in the .rst file. We should purge unused files from Doc/includes/sqlite3. Separate PR?

Reply:
We can remove stale examples in a later PR and use the same BPO issue.

Reply:
All right, I'll prepare a PR. Thanks, Berker!

Reply:
Purge PR prepared: #26234